docs/coverage-reporter/index.md

```diff
@@ -289,7 +289,8 @@ Follow these instructions to validate that your coverage setup is working correc
             Coverage was uploaded for a commit that belongs to a branch that isn't analyzed by Codacy.
         </td>
         <td>
-            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch or target branch for pull requests is enabled on Codacy</a>.</p>
+            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>.</p>
+            <p>Alternatively, make sure that a pull request for the branch is open and that the target branch is enabled on Codacy.</p>
```

Comment:
The PR can also be a draft, but we refer to it as "open".

I guess we can keep the wording; the users can figure this out on their own.

Reply:
Yes, I agree - we can talk about "opening a pull request" without specifying if it's in the **Draft** or **Open for review** state. :+1: